sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/models/ExtractSummaryAction.java

```diff
@@ -0,0 +1,108 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.ai.textanalytics.models;
+
+import com.azure.core.annotation.Fluent;
+
+/**
+ * Configurations that allow callers to specify details about how to execute an extractive summarization action in a
+ * set of documents.
+ */
+@Fluent
+public final class ExtractSummaryAction {
+    private String modelVersion;
+    private int maxSentenceCount;
+    private SummarySentencesOrder sentencesOrderBy;
```

Comment:
sentencesOrderBy -> sentenceOrder?

Reply:
I found Search is using getOrderBy() mehtod, and agreed to other languages to use the same name for now. But it is open to discuss in Java crews.